DemoServer/AdditionalLanguages/python/relatedDocuments/indexRelatedDocuments/indexRelatedDocuments.py

```diff
@@ -0,0 +1,45 @@
+#region Usings
+from ravendb import AbstractIndexCreationTask
+#endregion
+
+from demo_example import Example, RunParamsBase
+from models import Product
+from typing import List
+
+class RunParams(RunParamsBase):
+    def __init__(self, category_name: str):
+        self.category_name = category_name
+
+#region Demo
+#region Step_1
+class Products_ByCategoryName(AbstractIndexCreationTask):
```

Comment:
@poissoncorp 

pls let's talk about the following:

1. I removed the original enclosing class here
2. I think we need an inner 'IndexEntry' class - similar to other demos